src/corewlan.cs

```diff
@@ -544,4 +544,79 @@ interface CWNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopyi
 	interface CWMutableNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopying  
 	{
 	}
+
+	[Mac (10,10)] 
+	[BaseType (typeof (NSObject))]
+	interface CWWiFiClient
+	{
+		[Export ("delegate", ArgumentSemantic.Weak)]
+		[NullAllowed]
+		NSObject WeakDelegate { get; set; }
+
+		[Wrap ("WeakDelegate")]
+		[Protocolize]
+		CWEventDelegate Delegate { get; set; }
```

Comment:
Since this is a new binding I think we could just avoid the Weak Delegate here and do

```
interface ICWEventDelegate { }

[Mac (10,10)] 
[BaseType (typeof (NSObject))]
interface CWWiFiClient
{
	[Export ("delegate", ArgumentSemantic.Weak)]
	[NullAllowed]
	ICWEventDelegate Delegate { get; set; }
// ...
}
```

So no need for the WeakDelegate and the Protocolize attribute this way